otx/cli/utils/parser.py

```diff
@@ -78,7 +84,7 @@ def _get_default_metavar_for_optional(self, action):
         return action.dest.split(".")[-1].upper()
 
 
-def add_hyper_parameters_sub_parser(parser, config, modes=None):
+def add_hyper_parameters_sub_parser(parser, config, modes=None, return_sub_parser=False):
```

Comment:
How about just making this function always return and remove `return_sub_parser` argument?

Reply:
That would be nice too, but this is currently only used in train.py.

Reply:
Yes, you're right. But others which use this function just don't use return value then. Actually, this is a one of design choices. So if you think current implementation is better than just let it be and resolve this conversation.

Reply:
Yes your suggestion is good but doing so requires a lot of changes.